py-polars/src/dataframe.rs

```diff
@@ -625,7 +622,7 @@ impl PyDataFrame {
                 .has_header(has_header)
                 .with_delimiter(separator)
                 .with_line_terminator(line_terminator)
-                .with_quoting_char(quote)
+                .with_quoting_char(quote_char)
```

Comment:
`.with_delimiter` should be renamed to `.with_separator`, and `.with_quoting_char` should be renamed to `.with_quote_char`.